justfile

```diff
@@ -39,21 +47,28 @@ load-cloudbeat-image kind='kind-multi':
 build-opa-bundle:
   mage BuildOpaBundle
 
-build-binary:
-  GOOS=linux go mod vendor
-  GOOS=linux go build -v
-
-build-cloudbeat: build-opa-bundle build-binary
+# Builds cloudbeat binary
+# Set GOOS and GOARCH to the desired values (linux|darwin, amd64|arm64) by default, it will use local platform
+build-binary $GOOS=LOCAL_GOOS $GOARCH=LOCAL_GOARCH:
+  @echo "Building cloudbeat binary for $GOOS/$GOARCH"
+  go mod vendor
+  go build -v
+
+# Builds cloudbeat docker image with the OPA bundle included
+# Set GOOS and GOARCH to the desired values (linux|darwin, amd64|arm64)
+build-cloudbeat $GOOS=LOCAL_GOOS $GOARCH=LOCAL_GOARCH: build-opa-bundle
+  build-binary $GOOS $GOARCH
   docker build -t cloudbeat .
```

Comment:
@oren-zohar, cloudbeat and docker shall use the same ARCH. Since you can provide ARCH, need to also to update docker build command.

Reply:
nice catch :kiss: